src/transaction/utils.ts

```diff
@@ -412,7 +417,136 @@ function convertMemo(memo: Memo): { Memo: ApiMemo } {
   };
 }
 
+/**
+ * Autofills fields in a transaction.
+ *
+ * @param {Transaction} tx A transaction to autofill fields.
+ * @param {Client} client A client.
+ * @returns {Transaction} An autofilled transaction.
+ */
+async function autofillTransaction(
+  tx: Transaction,
+  client: Client
+): Promise<Transaction> {
+  validateAccountAddress(tx, 'Account', 'SourceTag');
+  if (tx['Destination']) {
+    validateAccountAddress(tx, 'Destination', 'DestinationTag');
+  }
+
+  // DepositPreauth:
+  convertToClassicAddress(tx, 'Authorize');
+  convertToClassicAddress(tx, 'Unauthorize');
+  // EscrowCancel, EscrowFinish:
+  convertToClassicAddress(tx, 'Owner');
+  // SetRegularKey:
+  convertToClassicAddress(tx, 'RegularKey');
+
+  setTransactionFlagsToNumber(tx);
+
+  if (tx.Sequence === undefined) {
```

Comment:
```suggestion
  if (tx.Sequence == null) {
```
We want to avoid using `undefined` to be slightly more general